### ISSUE TITLE
displayNames Ranomly changing 

### ISSUE THREAD

**Issue by LazzaAU** [project-alice-assistant/ProjectAlice#383]
My devices randomly change their device.displayName to what ever the device.typeName is . (so does phillips and other peoples devices so not related to just HA skill)

EG: displayName which should be "sink light" becomes "HAlight"

 This is very intermittent as i can go several restarts without issue but then several restarts in a row where it keeps popping up.
 
I feel it has something to do with the vicinity of 
```
def _loadConfigs(self):
		self._deviceConfigs.setdefault('displayName', self._data.get('displayName', self._typeName))
```
I say that for a couple of reasons:

1. self._data.get('displayName', self._typeName)) is what it's doing making displayName "typeName"
2. It ONLY changes on startup... does not change the value on shutdown

**Comment by Psychokiller1888** [comment]
The loading is straigthforward

deviceConfigs['displayName'] is replaced, if it is not yet existing, with _data['displayName'] if existing, else the typename

The quest here is then, why displayname doesn't exist in data for Alice to use the type name

**Comment by LazzaAU** [comment]
yes agreed, something i cant find atm

**Comment by LazzaAU** [comment]
fixed with #391
